pages/_app.js

```diff
@@ -106,6 +106,15 @@ function Header() {
               <WalletMultiButton />
             ) : (
               <div className={"user-wallet"}>
+                <Link href="#" className={"select-chain"} onClick={() => setShowModal(true)}>
```

Comment:
I had asked you to remove these attempts to make the button work because they are all broken